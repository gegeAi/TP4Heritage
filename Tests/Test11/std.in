R rec 1 2 3 4
DELETE rec seg
LIST
EXIT
