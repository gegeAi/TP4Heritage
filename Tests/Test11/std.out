OK
#New object: rec
R rec [1;2] [3;4]
